src/components/Splash.js

```diff
@@ -132,6 +133,20 @@ margin-bottom: 1rem;
 }
 `
 
+const GuildCardRootStyle = {
+  minWidth: `256px`,
+}
+
+const GuildColumnStyle = createGlobalStyle`
```

Comment:
Would it be possible to use a `styled-component` for this instead?

```suggestion
const EventColumn = styled.div`
```